dist/migration/migrate-page/ab-migrate.php

```diff
@@ -0,0 +1,69 @@
+<?php
+/**
+ * Getting Started content for Atomic Blocks
+ *
+ * @package Atomic Blocks
+ */
+
+?>
+
+<div class="wrap ab-migrate">
+	<div class="intro-wrap">
+		<div class="intro">
+			<a href="<?php echo esc_url( 'https://goo.gl/NfXcof' ); ?>"><img class="atomic-logo" src="<?php echo esc_url( plugins_url( 'getting-started/images/logo.png', dirname( dirname( __FILE__ ) ) ) ); ?>" alt="<?php esc_html_e( 'Visit Atomic Blocks', 'atomic-blocks' ); ?>" /></a>
+			<h1><?php echo esc_html__( 'Get ready for', 'atomic-blocks' ); ?> <strong><?php echo esc_html__( 'Genesis Blocks', 'atomic-blocks' ); ?></strong></h1>
+		</div>
+	</div>
+
+	<div class="panels">
+		<div id="panel" class="panel">
+			<div id="atomic-blocks-panel" class="panel-left visible">
+				<div class="ab-block-split clearfix">
+					<div class="ab-block-split-left">
+						<div class="ab-titles">
+							<h2><?php esc_html_e( 'Atomic Blocks has moved!', 'atomic-blocks' ); ?></h2>
+							<p>
+							<?php
+								/* translators: %1$s Genesis Pro website URL. */
+								echo sprintf( esc_html__( 'Same powerful blocks, same beautiful designs, same innovative team. Atomic Blocks been renamed to Genesis Blocks. %1$s', 'atomic-blocks' ), '<a href="https://studiopress.com/genesis-blocks/">Learn more about Genesis Blocks.</a>' );
+							?>
+							</p>
+							<?php
+							if ( version_compare( PHP_VERSION, '7.1' ) >= 0 ) {
```

Comment:
```suggestion
							if ( PHP_VERSION_ID >= 70100 ) {
```

Similar to how we do [here](https://github.com/studiopress/atomic-blocks/blob/edfa84de5d28100e95a3fe10d5341bc99378ecc9/loader.php#L45)... checks against the `PHP_VERSION_ID` constant, which is always an integer and doesn't require `version_compare()`.